modules/phase_field/src/kernels/KKSPhaseChemicalPotential.C

```diff
@@ -32,6 +32,14 @@ validParams<KKSPhaseChemicalPotential>()
                                                 "Base name of the free energy function "
                                                 "Fb (f_name in the corresponding "
                                                 "derivative function material)");
+  params.addParam<Real>("ka",
```

Comment:
Please update the KKS model derivations documentation page for the new capability